Libs/vtkSegmentationCore/vtkBinaryLabelmapToClosedSurfaceConversionRule.cxx

```diff
@@ -319,7 +388,7 @@ bool vtkBinaryLabelmapToClosedSurfaceConversionRule::CreateClosedSurface(vtkOrie
     }
 
   // Decimate
-  if (decimationFactor > 0.0)
+  if (decimationFactor > 0.0 && generationMethod == 0)
```

Comment:
```suggestion
  if (decimationFactor > 0.0)
```

See https://github.com/Slicer/Slicer/pull/7224#issuecomment-1737997911

Reply:
It would be nice to have enums instead of raw numbers here and in the test scripts.